packages/tradershub/src/components/ActionScreen/__tests__/ActionScreen.spec.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import ActionScreen from '../ActionScreen';
+
+jest.mock('@deriv/quill-design', () => {
+    return {
+        qtMerge: jest.fn(),
+    };
+});
+jest.mock('@deriv-com/ui', () => {
+    return {
+        Text: jest.fn(({ children }) => <span>{children}</span>),
+    };
+});
```

Comment:
NOTE: have some troubles with testing components from quill-design package, have the next error `SyntaxError: Cannot use import statement outside a module`. Trying to find a solution in jest.config.js.

Reply:
you can add these config to your package's jest.config file
![image](https://github.com/binary-com/deriv-app/assets/135801848/7d4d40df-b0cf-4e54-81d2-ce51708008eb)
'\\.css$': '<rootDir>/../../__mocks__/styleMock.js',
transformIgnorePatterns: ['/node_modules/(?!(@deriv/quill-design|@deriv-com/ui)).+\\.js$'],
I was also facing some issue in quill-design, that's why added it.

Reply:
@arshad-rao-deriv Tried to add this to jest.config and it didn't work for me... Will try with css too. Thank you

<img width="1093" alt="image" src="https://github.com/binary-com/deriv-app/assets/120570511/2d7bd6dc-d9ee-4851-be5a-ee979f8bd5a3">

Reply:
![image (11)](https://github.com/binary-com/deriv-app/assets/120570511/614dd10a-d352-4ccf-b509-40f5e7341e74)